### PR TITLE
Register cfp.is-a.dev and scfp.is-a.dev back

### PR DESCRIPTION
**Pull request by cfpwastaken**
Added `cfp.is-a.dev` and `scfp.is-a.dev` using the [dashboard](https://manage.is-a.dev) and manual edits.

### CODE REVIEW

**Comment by cfpwastaken** [comment]
I changed the files manually, cos the dashboard didnt give me the option to customize the owner object.

The screenshots are blank because my nginx automatically redirects to cfp.is-a.dev.

File: domains/cfp.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "cfpwastaken",
+        "email": "",
+        "discord": "cfp"
+    },
+    "record": {
+        "CNAME": "cfp.gotdns.ch"
```
Comment (DEV-DIBSTER) [inline domains/cfp.json:8]: Unable to access CNAME.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by YouFoundAlpha — CHANGES_REQUESTED** [review]
Please add your Discord username + Discord tag in the Discord info or add your Discord user id

**Comment by cfpwastaken** [comment]
Added discord id

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**